libsyclinterface/helper/source/dpctl_error_handlers.cpp

```diff
@@ -40,6 +41,29 @@ void DPCTL_AsyncErrorHandler::operator()(
     }
 }
 
+class GLogHelper
```

Comment:
Move GLogHelper into anonymous namespace.

Reply:
In general, any symbol you do not want to leak outside the translation unit should be in an anonymous namespace.